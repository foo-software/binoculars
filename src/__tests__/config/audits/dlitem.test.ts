import auditDlitem from '../../../config/audits/dlitem';

describe('dlitem audit', () => {
  it('should match snapshot', () => {
    expect(auditDlitem('en')).toMatchSnapshot();
  });
});
