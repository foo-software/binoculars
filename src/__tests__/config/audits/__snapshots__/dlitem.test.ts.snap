// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dlitem audit should match snapshot 1`] = `
Object {
  "description": "Definition list items (\`<dt>\` and \`<dd>\`) must be wrapped in a parent \`<dl>\` element to ensure that screen readers can properly announce them. [Learn more](https://web.dev/dlitem/).",
  "title": "Definition list items are wrapped in \`<dl>\` elements",
}
`;
